.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.5.1
```

Comment:
The ruby- is unnecessary and may interfere with version managers other than rvm and rbenv

```suggestion
2.5.1
```